build-system/common/github-actions.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/**
+ * @fileoverview Provides functions that extract various kinds of Github Actions
+ * state.
+ */
+
+/**
+ * Returns true if this is a Github Actions build.
+ * @return {boolean}
+ */
+function isGithubActionsBuild() {
```

Comment:
nit: GitHub

Reply:
I initially used the proper stylized name, but discovered that camel case rules require that only the first character of each word be capitalized. I've fixed the comments though. 😃